MVCDemos/Views/DropDownButton/Overview.cshtml

```diff
@@ -148,10 +148,7 @@
 
     function applyColor($element, color) {
         if (color) {
-            $element.removeClass("dx-theme-text-color");
             $element.css("color", color);
-        } else {
-            $element.addClass("dx-theme-text-color");
```

Comment:
If color is not defined, we should remove inline style from $element, so you code should include else section with clear current color style property.

Reply:
this is the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/9e5cda76dc0f3d4c2790f1dfb878df4a97708710)